src/include/iscsi-lio-server/widgets.rb

```diff
@@ -341,7 +341,7 @@ def LUNDetailDialog(pos, items)
         InputField(
           Id(:name),
           Opt(:hstretch),
-          "Name (autogenerated when empty):",
+          _("Name (autogenerated when empty):"),
```

Comment:
Note: usually the widget labels do not use the trailing `:`, remove it also from the other labels (`Path:`).